engines/stark/services/userinterface.cpp

```diff
@@ -489,7 +492,13 @@ void UserInterface::handleKeyPress(const Common::KeyState &keyState) {
 	} else if (keyState.keycode == Common::KEYCODE_F7) {
 		toggleScreen(Screen::kScreenSettingsMenu);
 	} else if (keyState.keycode == Common::KEYCODE_F8) {
-		warning("TODO: Implement the screenshot saving to local game directory");
+		OSystem_SDL *g_systemSdl = dynamic_cast<OSystem_SDL*>(g_system);
```

Comment:
Sorry, I missed that code. You really shouldn't call the SDL backend like this here. Our Android backend does not use SDL, and we can add more non-SDL backends in the future. The proper way to do that is to add a `saveScreenshot` to the `OSystem` class that backends can implement.